.github/workflows/multus.yaml

```diff
@@ -63,14 +60,47 @@ jobs:
         run: kubectl create -f tests/scripts/multus/default-public-cluster-nads.yaml
 
       - name: Quickly build Rook binary
+        id: build
         run: go build -o rook cmd/rook/*.go
 
-      - name: Run multus validation test
-        run: |
-          export KUBECONFIG="/home/runner/.kube/config"
-          kubectl create namespace rook-ceph
-          ./rook --log-level debug multus validation run \
-            --namespace rook-ceph \
-            --public-network default/public-net \
-            --cluster-network default/cluster-net \
-            --daemons-per-node 2
+      - name: Run CLI validation test
+        run: ./tests/scripts/multus/101-cli-test.sh
+
+      - name: Label kind nodes for stretch cluster tests
+        id: label
+        # tests that require labels are independent from previous tests
+        if: steps.build.outcome == 'success' && !cancelled()
```

Comment:
This condition is so the test will still run even if previous tests failed? Should we revert it before merge?

Reply:
Or is this condition saying to skip the test if a previous step failed? If that's the case, I thought that behavior was implicit without a condition like this.

Reply:
This, and the other `if` conditions like it, are so that the different test scripts can all run even if one flakes. I'm hoping that it'll allow us to manually identify individual flakes in that case. And in the case of a code bug, I'm hopeful it'll help get more information about failures across the various tests.

Reply:
Ok, so the intention is to have all test steps run even if one of them fails. Sounds good to keep it, assuming the overall test still fails.